Jint/Native/Array/ArrayPrototype.cs

```diff
@@ -383,22 +382,27 @@ private JsValue Splice(JsValue thisObj, JsValue[] arguments)
             var actualDeleteCount = System.Math.Min(System.Math.Max(TypeConverter.ToInteger(deleteCount), 0), len - actualStart);
             for (var k = 0; k < actualDeleteCount; k++)
             {
-                var from = (actualStart + k).ToString();
+                var from = TypeConverter.ToString(actualStart + k);
                 var fromPresent = o.HasProperty(from);
                 if (fromPresent)
                 {
                     var fromValue = o.Get(from);
-                    a.DefineOwnProperty(k.ToString(), new PropertyDescriptor(fromValue, true, true, true), false);
+                    a.DefineOwnProperty(TypeConverter.ToString(k), new PropertyDescriptor(fromValue, true, true, true), false);
                 }
             }
 
-            var items = arguments.Skip(2).ToArray();
```

Comment:
removeing Linq version which can be notoriously bad, especially on full framework where .ToArray is not optimized to predict wanted size